CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## v7.4.0
+
+Features:
+
+- Allow force-enabling or force-disabling colorized output
```

Comment:
We should really start styling the changelog writing with the past tense or present simple with references to comparisons with the previous release. Like using phrases along the lines of "X now does Y". The current content reminds a Git log rather than the release notes..